critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -56,6 +56,7 @@
     {% block license_agreement_input %} {% endblock %}
   </form>
   {% block buttons %} {% endblock %}
+  <br /><small class="text-warning"><em>{{ _('Note: Drafts are not private!') }}</em></small>
```

Comment:
Is this from another pull request? You should keep each in separate branches based on *master*.